### PR TITLE
Fix build error with xcode 9

### PR DESCRIPTION
**Pull request by yongpingchen**
- fixed xib file to build for iOS 8.0 or later
- fixed appledoc build failure. 
  - remove appledoc binary
  - use latest appledoc
  - fixed document warning caused appledoc build failure. 
- GHUnit was deprecated, so skip the test in CI. 
  - ios-sim launch GHUnit test app, even the tests finished, the command can not exit. (https://github.com/phonegap/ios-sim/issues/201)
  - all of the tests were passed. (already check locally)

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -1,14 +1,16 @@
 machine:
   xcode:
-    version: "7.2"
+    version: "9.2.0"
 
 dependencies:
   pre:
     - brew install ios-sim
+    - ios-sim showdevicetypes
```
Comment (dairyo) [inline circle.yml:8]: Why do we need `ios-sim showdeivetypes`?

**Review by dairyo — COMMENTED** [review]
I have checked https://github.com/phonegap/ios-sim/issues/201 but I could not find why ios-sim command does not exits.

Could you explain detail of it?

File: circle.yml (hunk 1 above)
Comment (yongpingchen) [reply circle.yml:8]: want to confirm the list of supported device in circle ci. Since the list is different with local ones.

**Comment by yongpingchen** [comment]
@shinsuke-nara 
> I have checked phonegap/ios-sim#201 but I could not find why ios-sim command does not exits.
> Could you explain detail of it?

`exist` is `exit`, right? 
I also don't know why the command not exit. But it did happen in my machine and circle CI. 
I guess it caused by GHUnit tests. 
And since GHUnit tests were deprecated. We don't want to spend more time on them.

File: circle.yml (hunk 1 above)
Comment (dairyo) [reply circle.yml:8]: I see.
Can you remove this?

**Comment by dairyo** [comment]
@yongpingchen 

https://github.com/KiiPlatform/photocolle-iOSSDK/blob/f81a9810fdb976c3e543b0d546df2c988cbda624/photocolle-sdk/Makefile#L34

In develop branch, we use `--exit` option for ios-sim. Could you try this?

**Comment by yongpingchen** [comment]
develop branch already fixed it. We decide to close it

**Pull request closed without merging**